kill.c

```diff
@@ -73,6 +73,9 @@ int kill_wait(const poll_loop_args_t* args, pid_t pid, int sig)
     }
     meminfo_t m = { 0 };
     const unsigned poll_ms = 100;
+    if (args->kill_process_group) {
+        pid = -getpgid(pid);
```

Comment:
Error handling, please! getpgid() can fail.

Reply:
Sure. I'm not sure how the error code in `fatal()` is chosen so for now I just randomly picked 7.